### PR TITLE
Heather b db connection

### PR DESCRIPTION
**Pull request by vbchomp**
added fix for db_url, comments and sample.env file.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,18 +4,15 @@
...
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Floof-Finders/lost-pet-finder-backend.git"
   },
-  "author": {
-    "name": "mark t", 
-    "name": "charlie f", 
-    "name": "vbchomp" 
-  },
+  "author": "markjackson28, fadnesscharlies, vbchomp",
```
Comment (fadnesscharlie) [inline package.json]: fadnesscharlie Bro!

**Review by markjackson28 — APPROVED** [review]
Looks good!

**Review by fadnesscharlie — APPROVED** [review]
I approve of these changes

**Review by fadnesscharlie — APPROVED** [review]
Clicked on the files viewed

**Pull request merged**